superset/db_engine_specs/postgres.py

```diff
@@ -45,6 +48,28 @@ class PostgresBaseEngineSpec(BaseEngineSpec):
     engine = ""
     engine_name = "PostgreSQL"
 
+    column_type_mappings = (
```

Comment:
Would it make sense to consolidate the structure of `db_column_types` and `column_type_mappings` as they are basically doing very similar things (one to convert db column type to generic types; one to convert to SQLA types)?

Reply:
Yes - this is the ultimate objective. However, we want to leave the existing structures untouched until we're certain this doesn't break existing functionality. We'll be including other types in this mapping that are missing in `db_column_types`, like `DbColumnType`, PyArrow types etc to make sure this is a one-stop-shop for making available all necessary types in one place.